manifests/charts/istiod-remote/templates/istiod-injector-configmap.yaml

```diff
@@ -0,0 +1,33 @@
+{{- if not .Values.global.omitSidecarInjectorConfigMap }}
```

Comment:
same comment about the injection template

Reply:
same comment about resolve. I understand how much simpler this would be to deliver as a file in the `samples` directory.